documenting/automation/solution-pack/SP-Doc-Automation/utils.py

```diff
@@ -0,0 +1,34 @@
+import zipfile
+import os
+import logging
+
+
+class Utility:
+
+    def __init__(self, path):
+        self.path = path
+
+    def unzip_zip_folder(self):
+        with zipfile.ZipFile(self.path, 'r') as unzip_path:
+            unzip_path.extractall(path=os.path.dirname(unzip_path.filename))
+            unzipped_folder_path = os.path.join(os.path.dirname(unzip_path.filename), os.path.splitext(os.path.basename(self.path))[0])
+        logging.debug('Successfully unzip Solution Pack')
+        return unzipped_folder_path
+        # with ZipFile(self.sp_path, 'r') as zip_ref:
+        #     # Extract all contents to the same directory as the zip file
+        #     extract_path = os.path.dirname(zip_ref.filename)
+        #     zip_ref.extractall(path=extract_path)
+        #
+        #     # Get the expected directory name
+        #     base_name = os.path.splitext(os.path.basename(self.sp_path))[0]
+        #     expected_dir_name = base_name.split(' ')[0]
```

Comment:
commented code. Please remove, if not needed

Reply:
It might be required in upcoming version